src/App.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { BrowserRouter, Routes, Route } from 'react-router-dom';
+import Navbar from './components/Navbar/Navbar';
+import Home from './pages/Home/Home';
+import CountryDetail from './pages/CountryDetail/CountryDetail';
+import './App.css';
+
+function App() {
+  return (
+    <BrowserRouter>
+      <Navbar />
+      <Routes>
+        <Route path="/" element={<Home />} />
+        <Route path="/countrydetail" element={<CountryDetail />} />
```

Comment:
- [ ]  All category details has the same URL, You should make sure that the category item detail page has its unique route within The SPA.  I recommend to use [usePrams](https://reactrouter.com/en/main/hooks/use-params) hook to fix the issue. This will make you project follow the Principle of Single Page Application.

Reply:
Thank you @ishimwezachee for your time and review. I have implemented the requested changes.